src/main/java/frc/robot/subsystems/Shooter.java

```diff
@@ -1,33 +1,54 @@
 /* ==================================================
- * Authors:
+ * Authors: Jason Wang
  *
  * --------------------------------------------------
- * Description:
- *
+ * Description: Shooter.java contains the functions to control the shooter.
  * ================================================== */
 
 package frc.robot.subsystems;
 
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import edu.wpi.first.wpilibj.motorcontrol.PWMTalonSRX;
 
 public class Shooter extends SubsystemBase
 {
   // HARDWARE //
-
-  // declare motor controllers, solenoids, and other hardware here
-
-  // CONSTANTS //
-
-
-
-  // CONSTRUCTOR //
-
+  private PWMTalonSRX motor;
+  public double speed;
+ // Ports //
+ public static final int SHOOTER_MOTOR = -1;
+ // Constants // 
+public static final double DEFAULT_SHOOTER_SPEED = 0;
```

Comment:
Let's set the default speed to 0.5 (1 might be overkill but we can always change it later)

Reply:
Actually, since we are gonna set the speed from the controller, we can just delete this constant.